### PR TITLE
Add input format verification

### PR DESCRIPTION
**Pull request by jayantk**
This PR adds an InputFormat class that you can specify with Inputs. The format will check if the entered value is of the appropriate type (hex string, etc). If not, it will return an error message that is displayed by the input.

The error messages interact a little weirdly with the input argument tables right now (they resize the table when they show up). Let's circle back on that when we get to styling things properly though.

### CODE REVIEW

File: contexts/GlobalContext.tsx [hunk 1]
```diff
@@ -31,8 +31,8 @@ export interface GlobalContextData {
 export type NetworkType = "mainnet" | "testnet";
 
 export const PriceServiceUrls: Record<string, string> = {
-  "mainnet": "https://xc-mainnet.pyth.network/",
-  "testnet": "https://xc-testnet.pyth.network/",
+  "mainnet": "https://xc-mainnet.pyth.network",
+  "testnet": "https://xc-testnet.pyth.network",
```
Comment (jayantk) [inline contexts/GlobalContext.tsx:35]: drive-by. the trailing slash was breaking the queries

File: components/EvmSend.tsx [hunk 2]
```diff
@@ -63,12 +63,10 @@ const EvmSend: React.FC<EvmSendProps> = ({
       const args: any[] | undefined = buildArguments(keyValueStore);
 
       let extraArguments: any | undefined = {};
-      let feeString = '';
```
Comment (jayantk) [inline components/EvmSend.tsx:66]: drive-by: field was unused

File: utils/InputFormat.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import {ethers} from 'ethers';
+
+// Checks whether the value of an input is a valid value.
+// Returns undefined if valid, or an error message string if invalid.
+export type InputFormat = (string) => string | undefined;
```
Comment (cctdaniel) [inline utils/InputFormat.tsx:5]: not a big deal but tslint was showing a warning here `Parameter has a name but no type. Did you mean 'arg0: string'?ts(7051)`

**Review by cctdaniel — APPROVED** [review]

File: utils/InputFormat.tsx (hunk 3 above)
Comment (jayantk) [reply utils/InputFormat.tsx:5]: ah... yeah i gotta set up ci. let me do that next.
